samples/msal-react-samples/b2c-sample/src/App.js

```diff
@@ -33,36 +30,6 @@ function App({ pca }) {
 }
 
 function Pages() {
-	const { instance } = useMsal();
```

Comment:
Could you please add a section to each of the FAQs with the relevant code samples if someone is using the legacy password reset flow? I would imagine many customers are still using the dedicated user flows and don't know to change over or don't want to. This information should still be available somewhere.

Reply:
Makes sense, will do 👍

Reply:
@tnorling done! React FAQ apparently already had this, and added one to browser FAQ as well. Will add to node as well once FAQ comes along.